cmd/install/install.go

```diff
@@ -107,6 +107,7 @@ type Options struct {
 	ExternalDNSDomainFilter                   string
 	ExternalDNSTxtOwnerId                     string
 	ExternalDNSImage                          string
+	OperatorEnvironmentVariables              map[string]string
```

Comment:
I would rename it as AdditionalOperatorEnvVars or something like that because there are already existing env vars in the current HO deployment.